### PR TITLE
Update message for InvalidInternalBookmark

### PR DESCRIPTION
**Pull request by v-pegao**
Related workitem is [here](https://ceapex.visualstudio.com/Engineering/_workitems/edit/108157/)

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/dotnet/docfx/pull/4977)

### CODE REVIEW

File: test/Microsoft.DocAsCode.Build.Engine.Tests/IncrementalBuildTest.cs [hunk 1]
```diff
@@ -1974,7 +1974,7 @@ public void TestCaseNotMatchIncludeFileWithInvalidBookmarkReplayLog()
                         templateFolder: templateFolder,
                         intermediateFolder: intermediateFolder);
                     Assert.Equal(2, Listener.Items.Count);
-                    Assert.NotNull(Listener.Items.FirstOrDefault(s => s.Message.StartsWith("Illegal link: `[link](#invalid)` -- missing bookmark")));
```
Comment (herohua) [inline test/Microsoft.DocAsCode.Build.Engine.Tests/IncrementalBuildTest.cs]: do we want to remove this?

Comment (v-pegao) [reply test/Microsoft.DocAsCode.Build.Engine.Tests/IncrementalBuildTest.cs]: @herohua It's better to check if the log item is what we want or not.

File: src/Microsoft.DocAsCode.Build.Engine/PostProcessors/ValidateBookmark.cs [hunk 2]
```diff
@@ -131,22 +131,18 @@ protected override Manifest PostHandleCore(Manifest manifest)
...
                         if (internalBookmark)
                         {
                             errorCode = WarningCodes.Build.InvalidInternalBookmark;
-                            dateWarning = " NOTE: This Suggestion will be elevated to a Warning on 8/26/2019. Please fix invalid bookmarks as soon as possible.";
+                            errorMessage = $"Invalid link: '{content}'. The file {linkedToFileSrc} doesn't contain a bookmark named '{bookmark}'.";
                         }
                         else
                         {
                             errorCode = WarningCodes.Build.InvalidExternalBookmark;
+                            errorMessage = $"Illegal link: `{content}` -- missing bookmark. The file {linkedToFileSrc} doesn't contain a bookmark named {bookmark}.";
```
Comment (superyyrrzz) [inline src/Microsoft.DocAsCode.Build.Engine/PostProcessors/ValidateBookmark.cs]: Could we use the same `errorMessage` for them? They are same before.

Comment (v-pegao) [reply src/Microsoft.DocAsCode.Build.Engine/PostProcessors/ValidateBookmark.cs]: @superyyrrzz Updated. They are using same log message like before.

**Comment by herohua** [comment]
@v-pegao , I guess the log codes should be the same as well?

File: test/Microsoft.DocAsCode.Build.Engine.Tests/IncrementalBuildTest.cs (hunk 1 above)
Comment (herohua) [reply test/Microsoft.DocAsCode.Build.Engine.Tests/IncrementalBuildTest.cs]: sorry I missed that. thanks for letting me know!

**Comment by v-pegao** [comment]
@herohua As confirmed with @superyyrrzz  yesterday, I will create an separate PR to merge the code `InvalidInternalBookmark` and `InvalidExternalBookmark`

**Pull request merged**